src/app/routes/login.$service.index.tsx

```
@@ -0,0 +1,81 @@
+import { createFileRoute } from '@tanstack/react-router';
+
+import { TOKEN_KEY } from '@/constants';
+import Layout from '@/shared/ui/layout/Layout';
+import { useEffect } from 'react';
+
+
+interface AuthResponse {
+  access_token: string;
+  token_type: string;
+  user: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)